scripts/docker_devboot.sh

```diff
@@ -71,6 +71,8 @@ run_master() {
 CFG_SITE_URL = u'http://localhost:5000'
 CFG_SITE_SECURE_URL = u'http://localhost:5000'
 CFG_REDIS_HOSTS = {'default': [{'db': 0, 'host': 'cache', 'port': 6379}]}
+CFG_DATABASE_NAME = u'invenio'
```

Comment:
Yes, but some overlays tend to overwrite these. And because we want to reuse the script for as many cases as possible, I enforce that default here.